### PR TITLE
feat: extend input component with disabled property

### PR DESCRIPTION
**Pull request by jonasbrunvoll**
Extend input component with disabled property

Skjermbilde 2024-11-13 kl  14 38 23
Skjermbilde 2024-11-13 kl  14 38 32

### CODE REVIEW

**Comment by mortennordseth** [comment]
Ref. discussion at the office. Need to take a look at the color choice here. Verify with sketches from the webshop, designers at AtB, etc.

File: src/page-modules/contact/components/form/form.module.css [hunk 1]
```diff
@@ -18,6 +18,10 @@
   gap: var(--spacings-small);
 }
 
+.label_disabled {
+  color: var(--text-colors-disabled);
+}
+
```
Comment (mortennordseth) [inline src/page-modules/contact/components/form/form.module.css:24]: Could this also be done with `: disabled` selector as you have done with cursor?

Comment (jonasbrunvoll) [reply src/page-modules/contact/components/form/form.module.css:24]: I dont think so. Is that possible when the <label> is outside <input/>?

Comment (mortennordseth) [reply src/page-modules/contact/components/form/form.module.css:24]: Oh, it's the label.

**Review by mortennordseth — APPROVED** [review]

**Pull request merged**